subsys/bluetooth/Kconfig.debug

```diff
@@ -0,0 +1,11 @@
+# Bluetooth debug configuration options
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_DEBUG_HCI_ERR_TO_STR
+	bool "Print HCI error codes as strings"
+	help
```

Comment:
can you add an `select EXPERIMENTAL`?
Then we can just merge your proposal and iterate after some real-world usage. I don't think this functionality is going to break much.

Reply:
I believe we should merge https://github.com/zephyrproject-rtos/zephyr/pull/75489 as part of the 3.7.0 release to avoid changing the API